Src/library/docking/implementation/controls/sd_title_bar.e

```diff
@@ -502,6 +503,76 @@ feature {NONE} -- Agents
 			end
 		end
 
+	on_dpi_change_fixed_resize (a_dpi, a_x: INTEGER_32; a_y: INTEGER_32; a_width: INTEGER_32; a_height: INTEGER_32)
```

Comment:
anyway to factorise code with on_fixed_resize feature??

Reply:
And this one ?

Reply:
Done.